shared/profile/showcase-team-offer/index.js

```diff
@@ -104,16 +115,16 @@ const ShowcaseTeamOffer = (props: Props) => (
         props.teamnames.map(name => (
           <TeamRow
             canShowcase={
-              props.teamNameToCanPerform[name] && props.teamNameToCanPerform[name].setMemberShowcase
+              (props.teamNameToRole[name] !== 'none' && props.teamNameToAllowPromote[name]) ||
+              ['admin', 'owner'].indexOf(props.teamNameToRole[name]) !== -1
```

Comment:
this extracts some logic from canUserPerform to safe us from calling that for every team. I can also move all this to `list` RPC.